using-mobile-extensions/adobe-edge-identity/README.md

```diff
@@ -0,0 +1,258 @@
+# Adobe Experience Platform Edge Identity
+The Adobe Experience Platform Edge Identity mobile extension enables identity management from your mobile app when using the [Adobe Experience Platform](https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/using-mobile-extensions/mobile-core) and the [Edge Network extension](https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/using-mobile-extensions/adobe-edge).
+
+## Configure the Adobe Experience Platform Edge Identity extension in Experience Platform Launch
+
+1. In Experience Platform Launch, in your mobile property, click the **Extensions** tab.
+2. On the **Catalog** tab, locate or search for the **Edge Identity** extension, and click **Install**.
+3. There are no configuration settings for **Edge Identity**.
+4. Click **Save**.
+5. Follow the publishing process to update SDK configuration.
+
+## Add the AEP Edge Identity extension to your app
+
+### Download and import the Edge Identity extension
+{% hint style="info" %}
+The following instructions are for configuring an application with Edge Identity. If an application will include both Identity and Edge Identity extensions, follow the instructions [below](#download-and-import-the-identity-and-edge-identity-extensions).
+{% tabs %}
+
+{% tab title="Android" %}
+### Java
+
+1. Add the Mobile Core and Edge extensions to your project using the app's Gradle file.
+
+   ```java
+   implementation 'com.adobe.marketing.mobile:core:1.+'
+   implementation 'com.adobe.marketing.mobile:edge:1.+'
+   implementation 'com.adobe.marketing.mobile:edgeidentity:1.+'
+   ```
+   
+2. Import the Mobile Core and Edge extensions in your application class.
+
+   ```java
+    import com.adobe.marketing.mobile.*;
+   ```
+{% endtab %}
+
+{% tab title="iOS" %}
+1. Add the Mobile Core and Edge extensions to your project using CocoaPods. Add following pods in your `Podfile`:
+
+   ```swift
+   use_frameworks!
+   target 'YourTargetApp' do
+       pod 'AEPCore'
+       pod 'AEPEdge'
+       pod 'AEPEdgeIdentity'
+   end
+   ```
+   
+2. Import the Mobile Core and Edge libraries:
+
+### Swift
+
+```swift
+// AppDelegate.swift
+import AEPCore
+import AEPEdge
+import AEPEdgeIdentity
+```
+
+### Objective-C
+
+```text
+// AppDelegate.h
+@import AEPCore;
+@import AEPEdge;
+@import AEPEdgeIdentity;
+```
+{% endtab %}
+{% endtabs %}
+
+### Register the Edge Identity extension with Mobile Core
+
+{% tabs %}
+{% tab title="Android" %}
+### Java
+
+```java
+public class MobileApp extends Application {
+
+    @Override
+    public void onCreate() {
+      super.onCreate();
+      MobileCore.setApplication(this);
+      MobileCore.configureWithAppID("yourLaunchEnvironmentID");
+
+      try {
+        Edge.registerExtension();
+        Identity.registerExtension();
+        MobileCore.start(new AdobeCallback() {
+          @Override
+          public void call(final Object o) {
+            // processing after start
+          }});
+      } catch (Exception e) {
```

Comment:
there is no exception thrown here.

Reply:
Don't we have the invalid init exception?

Reply:
Identity has it, but not Edge Identity.